### PR TITLE
feat: add side var menu

### PR DESCRIPTION
**Pull request by ManuViola77**
## What & Why:
This PR adds a side nav menu with options for different pages. 
Also, moved the logout button to the new settings page.

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/components/SideNav/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { NavLink } from 'react-router-dom';
+import KeyboardDoubleArrowLeftIcon from '@mui/icons-material/KeyboardDoubleArrowLeft';
+import KeyboardDoubleArrowRightIcon from '@mui/icons-material/KeyboardDoubleArrowRight';
+import cn from 'classnames';
+
+import { navData } from './navData';
+import './sidenav.css';
+
+const SideNav = () => {
+  const [open, setopen] = useState(false);
```
Comment (jpirazusta) [inline src/components/SideNav/index.js]:
```suggestion
  const [open, setOpen] = useState(false);
```

File: src/components/SideNav/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  const toggleOpen = () => {
+    setopen(!open);
```
Comment (jpirazusta) [inline src/components/SideNav/index.js]:
```suggestion
    setOpen(prev => !prev);
```
I think this is the most correct way of doing it

File: src/components/SideNav/sidenav.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
+.sidenav {
+  width: 250px;
+  transition: width 0.3s ease-in-out;
+  background-color: #538d4e;
+  padding-top: 28px;
+}
+
+.sidenav.closed {
+  transition: width 0.3s ease-in-out;
```
Comment (jpirazusta) [inline src/components/SideNav/sidenav.css]:
```suggestion
```
I think you don't need this here

File: src/components/SideNav/sidenav.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  visibility: hidden;
+}
+
+.sideitem:hover {
+  background-color: #244f7d1c;
+}
+
+.menuBtn {
+  align-self: center;
+  align-self: flex-start;
```
Comment (jpirazusta) [inline src/components/SideNav/sidenav.css]: 👀

**Review by jpirazusta — APPROVED** [review]
👏 👏

File: src/components/SideNav/sidenav.css (hunk 3 above)
Comment (ManuViola77) [reply src/components/SideNav/sidenav.css]: true! thanks!

**Pull request merged**